front/components/agent_builder/triggers/schedule/ScheduleEditionScheduler.tsx

```diff
@@ -0,0 +1,196 @@
+import {
```

Comment:
I'm wondering whether we could do something similar to what's done in `front/components/agent_builder/settings/AgentBuilderSettingsBlock.tsx` for the `AgentDescriptionInput`.